src/index.html

```diff
@@ -16,7 +16,94 @@
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo-wrapper">
+        <a href="index.html">
+          <img
+            src="images/logo.png"
+            alt="MOYO logo"
+          />
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item-1 is-active">
+            <a
+              class="nav__link nav__link-1"
+              href="index.html"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link nav__link-4"
+              href="index.html"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item nav__item-8">
```

Comment:
```suggestion
          <li class="nav__item nav__item">
```

Don't add extra classes, you can use :last-child rule to remove margin-right from the last element